docs/src/main/tut/docs/libraries/README.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
We might need a similar document like this for the `docs/rpc/README.md` path, where all the subsections above are included.

Reply:
I have one like this in RPC as well. Tonight I think that I will push the commit. Thanks for the reminder anyway.

Reply:
Pushed content : https://github.com/frees-io/freestyle-rpc/blob/35ebd55bb43fae28dcf927a65e438858f994ae51/docs/src/main/tut/README.md